packages/website/ts/translations/chinese.ts

```diff
@@ -0,0 +1,77 @@
+import { Key } from 'ts/types';
```

Comment:
can we make these language files normal json files? That way our internationalization pipeline be something like, new language gets added, then a json wtih all the translated strings gets created via dashboard or something, the dashboard writes json, and those files get imported into the source or pulled down by the app